operator/exporter.py

```diff
@@ -272,23 +272,23 @@ def collect_all_metrics():
             response = requests.get(
                 'http://'+ pod_details["ip_address"] +':8050/_api/metrics',
                 timeout=1)
+            if response.status_code == 200:
```

Comment:
- Ideally these statements will not raise an error if `response` is valid
- So, why not assign `None` to `response` before `try` and `return metrics` if `except` block is hit

Just a thought